layouts/partials/donate_two_percent.html

```diff
@@ -0,0 +1,14 @@
+<section class="hero is-warning is-small">
+	<div class="hero-body">
+		<div class="container">
+			<div class="columns has-text-centered-mobile">
+				<div class="column is-mobile-12">	
+					<h1 class="is-size-5-mobile is-size-4-tablet has-text-weight-bold">{{ .Params.redirect_20_title }}</h1>
+				</div>
+				<div class="column is-narrow">
+					<a class="button is-primary" target="_blank" rel="noopener noreferrer" href="https://code4.ro/{{ .Params.get_involved_url }}">{{ .Params.call }}</a>
```

Comment:
Pentru readability, `href` ar trebui să fie un link complet. Nu prea avem de ce să split it into domain în template și path în content